### PR TITLE
Webhook messages are stored

### PR DESCRIPTION
**Pull request by BertvanHoekelen**

### CODE REVIEW

File: migrations/2021_02_12_134612_create_webhook_notification_messages_table.php [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+use Illuminate\Support\Facades\Schema;
+
+class CreateWebhookNotificationMessagesTable extends Migration
+{
+    public function up(): void
+    {
+        Schema::create('webhook_notification_messages', function (Blueprint $table) {
+            $table->uuid('id');
+            $table->unsignedBigInteger('notifiable_id');
+            $table->string('notifiable_type');
```
Comment (huubvdw) [inline migrations/2021_02_12_134612_create_webhook_notification_messages_table.php]:
```suggestion
            $table->morphs('notifiable');
```
will also create an index on both columns

File: migrations/2021_02_12_134612_create_webhook_notification_messages_table.php [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+declare(strict_types=1);
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateWebhookNotificationMessagesTable extends Migration
+{
+    public function up(): void
```
Comment (huubvdw) [inline migrations/2021_02_12_134612_create_webhook_notification_messages_table.php:11]: Migrations should be publishable in the ServiceProvider.

**Review by huubvdw — CHANGES_REQUESTED** [review]

File: migrations/2021_02_12_134612_create_webhook_notification_messages_table.php (hunk 2 above)
Comment (BertvanHoekelen) [reply migrations/2021_02_12_134612_create_webhook_notification_messages_table.php:11]: That's done in the `LaravelWebhookNotificationServiceProvider`.

File: src/LaravelWebhookNotificationServiceProvider.php [hunk 3]
```diff
@@ -6,13 +6,26 @@
...
+use Spatie\WebhookServer\Events\WebhookCallSucceededEvent;
 
 class LaravelWebhookNotificationServiceProvider extends ServiceProvider
 {
     public function boot(): void
     {
         $this->app->make(ChannelManager::class)
             ->extend('webhook', fn (Application $app) => $app->make(WebhookChannel::class));
+
+        $this->loadMigrationsFrom(__DIR__ . '/../migrations');
```
Comment (BertvanHoekelen) [inline src/LaravelWebhookNotificationServiceProvider.php:23]: @huubvdw

File: migrations/2021_02_12_134612_create_webhook_notification_messages_table.php (hunk 1 above)
Comment (BertvanHoekelen) [reply migrations/2021_02_12_134612_create_webhook_notification_messages_table.php]: Done.

Comment (huubvdw) [reply migrations/2021_02_12_134612_create_webhook_notification_messages_table.php]: We only need one (composite) index, which is exactly what the `morphs` method does.

**Review by huubvdw — APPROVED** [review]

**Pull request merged**